omnibus/config/software/python2.rb

```diff
@@ -79,8 +79,8 @@
            :sha256 => "575093fd5748ccc22be6577fff15ae9ffe525b627888342bd43826053183e9da",
            :extract => :seven_zip
   else
-    source :url => "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-#{version}-nopip-amd64.zip",
-         :sha256 => "b9878cf2e64084c35a98ae1acd68f93bd7bc36e232e01088cba7692153068f67",
+    source :url => "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-#{version}-nopip-2-amd64.zip",
```

Comment:
Why changing python version?

Reply:
The python zip that shipped with the original 6.14 had extra CRT files in it, that would cause the agent to not be able to load python2 on win2k8.